.bazelrc

```diff
@@ -12,6 +12,7 @@ build --auto_cpu_environment_group=//buildenv:cpu
 
 # Ensure clang is used, by default, over any other C++ installation (e.g. gcc).
 # LLVM requires C++14, so ensure that is used.
+build --cxxopt=-std=c++14 --host_cxxopt=-std=c++14
```

Comment:
nit: add this on the same line as the BAZEL_CXXOPTS part?

Reply:
I'm happy to do that if you prefer: I didn't specifically because these are not "client environment" settings. But would you like me to combine all three lines, since they are all related to Clang settings? And: Would you like me to do this for the remote settings also?

Reply:
`BAZEL_CXXOPTS` doesn't matter for the remove config (because we disable the autoconfiguration), so don't do it there.

My suggestion was mostly around grouping the "C++14" settings together.

Reply:
Actually, it looks like the remote settings don't have the env settings anyway.

Reply:
Done (I think) PTAL.